CMakeLists.txt

```diff
@@ -287,7 +227,7 @@ endif()
 option(STATIC_EXECUTABLES "produce static executables" OFF)
 if (STATIC_EXECUTABLES)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static")
-  if (WINDOWS)
+  if (WIN32)
```

Comment:
since we actually don't build for 32 bit windows, I'd rather remove `WIN32` from the place instead of adding more

Reply:
`WIN32` in CMake doesn't make 32 bits.

From https://cmake.org/cmake/help/latest/variable/WIN32.html:

> Set to True when the target system is Windows, including Win64.